minigalaxy/ui/library.py

```diff
@@ -45,7 +45,6 @@ def __update_library(self):
         GLib.idle_add(self.__load_tile_states)
         # Get already installed games first
         self.games = self.__get_installed_games()
-        self.__create_gametiles()
```

Comment:
I don't think this line should be removed

Reply:
Yeah, I removed it because the store page URL is not available at the time where the tiles for installed games are created, and I noticed that create_gametiles is called later on anyway after self.games have been fully populated. The URL is read from the API in add_games_from_api but if the tile for a game already exists then it doesn't matter.

If you prefer to keep it as is I could take a second look. I think would need to find a way to update the tiles if they already exist in create_gametiles. The current behaviour is to skip the game if a tile for it already exists. The fixed behaviour would be to update the tile.

Reply:
This might be related to #110 then.

Reply:
Oh wow, I see it now. It is being called twice. Good catch!

Reply:
I just remembered the reason why I implemented it like this. If your internet connection is extremely slow, the tiles for the games you have installed will already show while. This is what is causing #110, but I think we should fix this is a different way. Can you revert this change?

Reply:
Ah I see. I reverted the removal.